### PR TITLE
Feat(eos_cli_config_gen): Add ipv6_attached_host_route_export for SVIs

### PR DESCRIPTION
**Pull request by emilarista**
## Change Summary

Adds data model for ipv6 attached-host route export command for vlan-interfaces.

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes

```yaml
vlan_interfaces:
  - name: vlanxyz
    ipv6_attached_host_route_export:
      enabled: <bool>
      distance: <int>
      prefix_length: <int>
```

## How to test

Tested with molecule and on device.

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/vlan-interfaces.cfg [hunk 1]
```diff
@@ -195,6 +195,24 @@ interface Vlan333
...
+   description v6 attached host exports
+   ipv6 enable
+   ipv6 address 2a00:2::1/64
+   ipv6 attached-host route export prefix-length 64
+!
+interface Vlan336
+   description v6 attached host exports
+   ipv6 enable
+   ipv6 address 2a00:3::1/64
+   ipv6 attached-host route export 18 prefix-length 64
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/vlan-interfaces.cfg]: This CLI line is rendered _above_ `ipv6 enable` on EOS. Please move it in the template.

**Review by ClausHolbechArista — CHANGES_REQUESTED** [review]

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vlan_interfaces.schema.yml [hunk 2]
```diff
@@ -491,6 +491,24 @@ keys:
...
+          keys:
+            enabled:
+              type: bool
+              required: true
+            distance:
+              type: int
+              convert_types:
+                - str
+              min: 1
+              max: 255
```
Comment (Shivani-gslab) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vlan_interfaces.schema.yml:505]:
```suggestion
              max: 255
              description: Administrative distance for this route.
```

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vlan_interfaces.schema.yml [hunk 3]
```diff
@@ -491,6 +491,24 @@ keys:
...
+            prefix_length:
+              type: int
+              convert_types:
+                - str
+              min: 0
+              max: 128
```
Comment (Shivani-gslab) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vlan_interfaces.schema.yml:512]:
```suggestion
              max: 128
              description: Length of generated host routes.
```

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/vlan-interfaces.cfg (hunk 1 above)
Comment (emilarista) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/vlan-interfaces.cfg]: Fixed, thanks!

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vlan_interfaces.schema.yml (hunk 2 above)
Comment (emilarista) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vlan_interfaces.schema.yml:505]: Added description, thanks!

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vlan_interfaces.schema.yml (hunk 3 above)
Comment (emilarista) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vlan_interfaces.schema.yml:512]: Fixed.

**Review by Shivani-gslab — APPROVED** [review]
LGTM!
(Need to resolve conflicts.)

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Pull request merged**